packages/vant/src/radio/index.less

```diff
@@ -56,12 +57,42 @@
       }
     }
 
+    &--dot {
+      position: relative;
+      border-radius: 100%;
+      box-sizing: border-box;
+      width: var(--van-radio-size);
+      height: var(--van-radio-size);
+      border: 1px solid var(--van-radio-border-color);
+      transition-duration: var(--van-radio-duration);
+      transition-property: border-color;
+
+      &__icon {
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        border-radius: 100%;
+        height: calc(100% - var(--van-radio-dot-size));
+        width: calc(100% - var(--van-radio-dot-size));
```

Comment:
Why it is `calc(100% - var(--van-radio-dot-size))` rather than `var(--van-radio-dot-size)`, does `--van-radio-dot-size` not represent the size of the inner dot icon?

Reply:
Oops, the variable name is a bit problematic(Thanks, I will improve it).  In fact, it's not about the size of the dot but rather the distance between the dot and the outer border.😶
![image](https://github.com/youzan/vant/assets/19986739/abb70095-7008-4c24-9f55-242504fce399)